### PR TITLE
Added email send changes, vendored gomail package

### PR DESCRIPTION
**Pull request by gejain**
**What this PR does / why we need it**:
- Added gomail package to use to send emails for torpedo longevity jobs
- Modified sendEmail function to use gomail
**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PTX-15093
**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Rohit-PX** [comment]
Please run torpedo dev job before to check for regression, check with Maksym/Leela as to which job that is.

File: pkg/email/email.go [hunk 1]
```diff
@@ -1,50 +1,37 @@
...
+	m := gomail.NewMessage()
+	m.SetHeader("From", email.From)
+	m.SetHeader("Subject", email.Subject)
+	tos := email.To
+	m.SetHeader("To", tos...)
+	m.SetBody("text/html", email.Content)
+	dialer := gomail.Dialer{Host: email.EmailHostServer, Port: EmailPortKey, Username: "", Password: ""}
+	dialer.TLSConfig = &tls.Config{InsecureSkipVerify: true}
+	if err := dialer.DialAndSend(m); err != nil {
+		log.Fatal(err)
```
Comment (lsrinivas-pure) [inline pkg/email/email.go:34]: return the error. Calling function will handle it

**Comment by gejain** [comment]
@Rohit-PX  - 
Please find the job run below -
https://jenkins.pwx.dev.purestorage.com/job/Dev/job/torpedo-dev01/205/
Aetos results - 
http://aetos.pwx.purestorage.com/resultSet/testSetID/88562

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Review by Rohit-PX — APPROVED** [review]

**Review by lsrinivas-pure — APPROVED** [review]

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by CharudathGopal — APPROVED** [review]

**Pull request merged**